core/src/main/java/com/netflix/conductor/core/execution/tasks/SetWorkflowVar.java

```diff
@@ -0,0 +1,39 @@
+package com.netflix.conductor.core.execution.tasks;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.Task.Status;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.execution.WorkflowExecutor;
+
+public class SetWorkflowVar extends WorkflowSystemTask {
+	public static final String NAME = "SET_WORKFLOW_VAR";
+	public SetWorkflowVar() {
+		super(NAME);
+	}
+
+	@Override
+	public boolean execute(Workflow workflow, Task task, WorkflowExecutor provider) {
```

Comment:
Curious as to why the `execute` method has been overridden to perform the update instead of `start` similar to an [Event](https://github.com/Netflix/conductor/blob/master/core/src/main/java/com/netflix/conductor/core/execution/tasks/Event.java#L65) task.